Presentation.Web/Tests/Helpers/Select2Helper.ts

```diff
@@ -0,0 +1,24 @@
+﻿import WaitTimers = require("../Utility/WaitTimers");
+
+class Select2Helper {
+    private static waitUpTo = new WaitTimers();
+    private static ec = protractor.ExpectedConditions;
+
+
+    public static waitForDataAndSelect() {
+        console.log(`waitForSelect2DataAndSelect`);
+        return browser.wait(this.ec.visibilityOf(element(by.className("select2-result-label"))), this.waitUpTo.twentySeconds)
+            .then(() => element(by.id("select2-drop")).element(by.className("select2-input")).sendKeys(protractor.Key.ENTER));
+    }
+
+    public static SearchFor(name: string, elementId: string) {
+        console.log(`select2SearchFor: ${name}, in element with id: ${elementId}`);
+        return element(by.id(elementId)).element(by.tagName('a')).click()
```

Comment:
Kan man ikke være lidt mere specifik end "a" ? .. det kan hurtigt blive skrøbeligt

Reply:
Måske by.className("select2-choice")

Reply:
just do it :-) "a" er i hvert fald lidt fattigt